microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-xml-maven.adoc

```diff
@@ -0,0 +1,73 @@
+= Microsoft .Net Connector 3.1 XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio.
+
+If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
```

Comment:
Why fully spell name here and not in first sentence instead? Because of template?